velox/connectors/hive/iceberg/tests/IcebergReadTest.cpp

```diff
@@ -106,6 +106,10 @@ class HiveIcebergTest : public HiveConnectorTestBase {
     // Keep the reference to the deleteFilePath, otherwise the corresponding
     // file will be deleted.
     std::vector<std::shared_ptr<TempFilePath>> deleteFilePaths;
+    // Use PARQUET instead of default DWRF if Parquet Writer Factory is present
```

Comment:
Nit: for comments please use a complete sentence with period at the end.
From the guidelines: "Comments should be full english sentences, starting with a capital letter and ending with a period (.).".

Here and elsewhere.